moxin-protocol/src/open_ai.rs

```diff
@@ -111,7 +121,7 @@ fn response_object() -> String {
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
 pub struct ChunkChoiceData {
-    pub finish_reason: StopReason,
+    pub finish_reason: Option<StopReason>,
```

Comment:
I noticed in the OpenAI documentation that it might be a null, so I changed it to an Option here.